cmd/otel-agent/config/agent_config.go

```diff
@@ -25,6 +25,40 @@ import (
 	"go.opentelemetry.io/collector/service"
 )
 
+type logLevel int
+
+const (
+	trace logLevel = iota - 1
+	debug
+	info
+	warn
+	err
+	critical
+	off
+)
+
+// datadog agent log levels: trace, debug, info, warn, error, critical, and off
+// otel log levels: disabled, debug, info, warn, error
```

Comment:
Will these values/mappings ever be changing over time? If so, will customers need to upgrade their otel agent to get accurate functionality? And if that would be a problem would we have a way to modify these via a config file instead of hard coded

Reply:
The log mapping reflects the agent mapping. It is not expected to change. If new options are added to the mapping they would be very unlikely to result in the removal of any of these from the map (though other new ones could be included). In any case, it would be rare to see these change anyhow.